Userland/Games/Snake/SnakeGame.h

```diff
@@ -54,6 +56,8 @@ class SnakeGame : public GUI::Frame {
     int m_rows { 20 };
     int m_columns { 20 };
 
+    int m_timer { 100 };
```

Comment:
Does this need to be a class variable? If the goal is to avoid magic numbers, a local `static constexpr` variable in `start()` would suffice

Reply:
makes sense, ty

Reply:
This needs to be removed now